ehr/src/org/labkey/ehr/table/DefaultEHRCustomizer.java

```diff
@@ -1080,9 +1081,9 @@ private void configureMoreActionsBtn(AbstractTableInfo ti, ButtonBarConfig cfg)
                     btnNameMap.put(newButton.getText(), newButton);
                     menuItems.add(newButton);
 
-                    for (ClientDependency cd : fact.getClientDependencies(ti.getUserSchema().getContainer(), ti.getUserSchema().getUser()))
+                    for (Supplier<ClientDependency> cd : fact.getClientDependencies(ti.getUserSchema().getContainer(), ti.getUserSchema().getUser()))
```

Comment:
Resolve through `ClientDependency.getClientDependencySet()`?

Reply:
Yes, thanks. That eliminates the concern about it resolving to null.